core/modules/widgets/draggable.js

```diff
@@ -67,6 +69,8 @@ DraggableWidget.prototype.execute = function() {
 	// Pick up our attributes
 	this.draggableTag = this.getAttribute("tag","div");
 	this.draggableClasses = this.getAttribute("class");
+	this.dragStartActions = this.getAttribute("startactions");
```

Comment:
I think we should use "startActions", "endActions" for consistency with e.g. LinkCatcher widget.

Reply:
Ok! :+1: